pyvista/core/filters/data_set.py

```diff
@@ -4154,15 +4154,20 @@ def plot_over_circular_arc_normal(
         if show:  # pragma: no cover
             plt.show()
 
-    def extract_cells(self, ind, progress_bar=False):
+    def extract_cells(self, ind, assume_unique=False, progress_bar=False):
         """Return a subset of the grid.
 
         Parameters
         ----------
-        ind : np.ndarray
+        ind : numpy.ndarray
             Numpy array of cell indices to be extracted.
 
-        progress_bar : bool, optional
+        assume_unique : bool, default: False
+            If the ``ind`` specified are already sorted and unique, then set
+            this to ``True`` to avoid the filter from doing time-consuming
+            sorts and uniquification operations.
+
+        progress_bar : bool, default: False
```

Comment:
Are we using this numpydoc language feature now?  I ask mainly for when I contribute and for reviewing.  My observation is that we use `optional` with default in the description throughout the code.  I personally prefer the style of `default: False`, but it does make the documentation a bit more inconsistent if the default is more complex and it has to be explained in the description.

Reply:
I agree it's inconsistent. If it's alright with you, we can leave it in this PR and have a follow-up before 0.37.0 and fix the docs. I agree that it's way better to use `default: value`